input/metadata.json

```
@@ -0,0 +1,1111 @@
+          },
+          "do_not_breed_time": {
+            "type": "number",
+            "description": "Do Not Breed Time (days) -- The breeding period (days) for cows before reproductive programs stop if they fail to get pregnant (days)",
+            "default": 400,
+            "minimum": 0
+          },
+          "cull_milk_production": {
+            "type": "number",
+            "description": "Cull Milk Production (kg/d) -- Minimum daily milk production for do-not breed cow to be culled (default is 22 kg/d)",
+            "default": 22,
+            "minimum": 0
+          },
+          "cow_times_milked_per_day": {
+            "type": "number",
+            "description": "Number of Times Cows Milked (per day) -- The average or most common number of times the cows are milked, whole numbers 1-3",
+            "default": 3,
+            "minimum": 0
+          },
+          "energy_and_nutrient_calculation_method": {
+            "type": "string",
+            "default": "NASEM",
+            "pattern": "^{NASEM|NRC}$",
+            "description": "Energy and Nutrient Calculation Method -- The method used to calculate the animal nutrient requirements.",
+          },
+        },
+        "farm_level": {
+          "type": "object",
+          "description": "Farm Level Management",
+          "calf": {
+            "type": "object",
+            "description": "Calf Management",
+            "male_calf_rate_sexed_semen": {
+              "type": "number",
+              "description": "Male Calf Rate - Sexed Semen -- The rate of male calves when using sexed semen (default is 0.1)",
+              "default": 0.1,
+              "minimum": 0,
+              "maximum": 1
+            },
+            "male_calf_rate_conventional_semen": {
+              "type": "number",
+              "description": "Male Calf Rate - Conventional Semen -- The rate of male calves when using conventional semen (default is 0.53)",
+              "default": 0.53,
+              "minimum": 0,
+              "maximum": 1
+            },
+            "keep_female_calf_rate": {
+              "type": "number",
+              "description": "Female Calf Rate -- The rate female calves are kept and raised on-farm (default is 1)",
+              "default": 1,
+              "minimum": 0,
+              "maximum": 1
+            },
+            "wean_day": {
+              "type": "number",
+              "description": "Wean Day (days) -- Days the calf is fully weaned from milk or milk replace (default is 56)",
+              "default": 56,
+              "minimum": 0
+            },
+            "wean_length": {
+              "type": "number",
+              "description": "Wean Length (days) -- Days weaning (default is 7)",
+              "default": 7,
+              "minimum": 0
+            },
+            "milk_type": {
+              "type": "string",
+              "default": "whole",
+              "pattern": "^{whole|replacement}$",
```

Sorry, got this wrong before, it should be `whole` or `replacer`.
```suggestion
              "pattern": "^{whole|replacer}$",
```